Cargo.lock

```diff
@@ -85,18 +85,28 @@ dependencies = [
 
 [[package]]
 name = "clap"
-version = "2.34.0"
+version = "3.0.1"
```

Comment:
Can we bump this again. There have been a couple of minor bugfix releases following the 3.0 release (3.0.5 as of writing).

Reply:
done